drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -110,6 +113,13 @@ class DragAndDropBlock(XBlock, XBlockWithSettingsMixin, ThemableXBlockMixin):
         default="",
     )
 
+    max_items_per_zone = Integer(
```

Comment:
@cahrens I've talked [about that]https://openedx.atlassian.net/browse/SOL-1979?focusedCommentId=206512&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-206512) and there were no objections to implementing this approach. So I would assume product is fine with single setting for all zones. 

@sstack22 - any comments? Just FYI, I'll be away starting Friday and I would really love to land all my PRs (or at least this and #93) before that; implementing per zone max items would make it problematic :)